domains/measurements/Measurements.API/API/Startup.cs

```diff
@@ -52,37 +39,15 @@ public void ConfigureServices(IServiceCollection services)
 
         services.AddEndpointsApiExplorer();
 
-        services.AddTransient<IConfigureOptions<SwaggerGenOptions>, ConfigureSwaggerOptions>();
+        services.AddSwagger("measurements");
         services.AddSwaggerGen();
```

Comment:
Shouldnt this be part of the nuget?

Reply:
No because that is used to register dokument filters (which is where have our api descriptions)